coding/url.hpp

```diff
@@ -18,7 +18,7 @@ class Url
   explicit Url(std::string const & url);
   static Url FromString(std::string const & url);
 
-  bool IsValid() const { return !m_scheme.empty(); }
+  bool IsValid() const { return !m_scheme.empty() && !m_host.empty(); }
```

Comment:
That's an interesting case, AFAIR now a URL with an empty host (e.g. `om://`) can launch the app without any parameters. Is it feasible to test how it works on a device with your changes?